index.js

```diff
@@ -118,7 +118,7 @@ function* run() {
       if (commander.bind_ip_all) {
         options.bind_ip_all = null;
       }
-      return options;
+      return {options: options};
```

Comment:
@vkarpov15 Did you check this line? I think this fix is needed and didn't go as part of #48 or #50

The way options is passed to ReplSet constructor is not in the format the Constructor expects it. That what i have fixed here.

Reply:
Oh my bad, I see you have changed options to { options } . That addresses it. Thanks.